python/lsst/ts/xml/data/sal_interfaces/LinearStage/LinearStage_Events.xml

```diff
@@ -2,6 +2,17 @@
 <?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
 <SALEventSet xmlns:xsi="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
   <Enumeration>DetailedState_NotMovingState, DetailedState_MovingState</Enumeration>
+  <Enumeration>
+    ErrorCode_ConnectionFailed,
```

Comment:
I see that the python enum explicitly defines the values of the enum. Please do that here as well like so:

```ErrorCode_ConnectionFailed=1,```

and also for the rest of the values of this enum.